docs/docs/40-contributor-guide/10-hacking-on-kargo.md

```diff
@@ -18,6 +18,13 @@ execute tests within a container that is maximally similar to those used in CI.
 
 To take advantage of this, you only need to have
 [Docker](https://docs.docker.com/engine/install/) and `make` installed.
+> you can use podman by creating an environment variable
+
+```bash
+export CONTAINER_RUNTIME=podman
+```
+
+[How to config podman socket](https://docs.podman.ioen/latest/markdown/podman-system-service.1.html#examples)
```

Comment:
Do we need to be podman-specific here?

Reply:
> Do we need to be podman-specific here?

I realized my meaning might have been ambiguous.

The comment _wasn't_ about the podman socket setup.

It was about the general idea of this PR adding _prodman_ support.

I'm fine with the notion of supporting alternative container runtimes as long as they are "Docker compatible," but I'm less ok with presenting podman as a specially preferred/specially supported alternative container runtime... if that makes any sense.

I think the easiest thing to do here might be to just not mention this in the docs. The changes to the `Makefile` are good and I think anyone who really cares about using an alternative container runtime will find the way of doing it easily enough.